### PR TITLE
Feature/validation various changes

### PR DESCRIPTION
**Pull request by ObiWahn**
https://github.com/arangodb/docs/pull/410 - documentation
https://github.com/arangodb/enterprise/pull/431 - enterprise part

### CODE REVIEW

File: arangod/Utils/Events.cpp [hunk 1]
```diff
@@ -33,6 +33,7 @@ void Authenticated(GeneralRequest const&, rest::AuthenticationMethod) {}
 void NotAuthorized(GeneralRequest const&) {}
 void CreateCollection(std::string const& db, std::string const& name, int result) {}
 void DropCollection(std::string const& db, std::string const& name, int result) {}
+void PropertyUpdateCollection(std::string const& collectionName, VPackSlice const& propertiesSlice) {}
```
Comment (jsteemann) [inline arangod/Utils/Events.cpp:36]: The collection name alone may not be useful when there are multiple database.
We should add the database name here too (similar to `TruncateCollection`, `DropCollection`, `CreateCollection`).

File: js/apps/system/_admin/aardvark/APP/frontend/js/models/arangoCollectionModel.js [hunk 2]
```diff
@@ -240,6 +240,33 @@
...
+          callback(true, data);
+        }
+      });
+      return result;
+    },
+
+    changeValidation: function (validation, callback) {
+      var result = false;
+      if (!validation) {
+        validation = {};
```
Comment (jsteemann) [inline js/apps/system/_admin/aardvark/APP/frontend/js/models/arangoCollectionModel.js:263]: shouldn't the default be `null` here, or doesn't this matter?

File: js/apps/system/_admin/aardvark/APP/frontend/js/routers/router.js [hunk 3]
```diff
@@ -1142,6 +1167,10 @@
       if (this.documentView && Backbone.history.getFragment().indexOf('collection') > -1) {
         this.documentView.resize();
       }
+      if (this.validationView && Backbone.history.getFragment().indexOf('cSchema') > -1) {
+        // TODO the resize does not work :( -- kittens die -- Heiko Help:)
+        this.validationView.resize();
```
Comment (jsteemann) [inline js/apps/system/_admin/aardvark/APP/frontend/js/routers/router.js:1172]: Ok, is that solved? We should address this before we merge.

**Comment by jsteemann** [comment]
http://172.16.10.101:8080/view/PR/job/arangodb-matrix-pr/9476/

**Comment by ObiWahn** [comment]
http://172.16.10.101:8080/view/PR/job/arangodb-matrix-pr/9488/

**Comment by ObiWahn** [comment]
http://jenkins.arangodb.biz:8080/view/PR/job/arangodb-matrix-pr/9494/

**Pull request merged**